examples/chip-tool/commands/tests/Commands.h

```diff
@@ -7829,6 +7832,75 @@ class TestCluster : public TestCommand
 
         runner->NextTest();
     }
+
+    // Test Send Test Command to unsupported endpoint
+    typedef void (*SuccessCallback_101)(void * context);
+    chip::Callback::Callback<SuccessCallback_101> * mOnSuccessCallback_101    = nullptr;
```

Comment:
Why does this not say:

```
chip::Callback::Callback<SuccessCallback_101> mOnSuccessCallback_101 { &OnTestSendClusterTestClusterCommandTest_101_FailureResponse, this };
```

And if it needs to be nullable for some reason (I'm not sure why..) we could use Optional<> or std::unique_ptr.

I think this is worse than all of 3 other possibilities.

Reply:
> Why does this not say:

I'm happy to look into this, but this is auto-generated code (see the two separate commits which break out the manual changes and the generated changes) so the fix would be pretty orthogonal and would affect all of the test commands, presumably.

Reply:
So there is no reason this _has_ to be nullable.  That said, right now we allocate the callback for a test right before we start that test, and deallocate it when the test finishes.  If we just made them non-nullable members we'd have them all allocated all the time, effectively.  That's probably not a problem for the sorts of devices that run chip-tool....